src/status_im/ui/components/bottom_sheet/view.cljs

```diff
@@ -0,0 +1,174 @@
+(ns status-im.ui.components.bottom-sheet.view
+  (:require [status-im.ui.components.react :as react]
+            [status-im.ui.components.animation :as animation]
+            [status-im.ui.components.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def initial-animation-duration 300)
+(def release-animation-duration 100)
+(def cancellation-animation-duration 150)
+(def swipe-opacity-range 100)
+(def cancellation-height 250)
+
+(defn animate-sign-panel
+  [opacity-value bottom-value]
+  (animation/start
```

Comment:
would this one use a native driver? otherwise it might be a bit choppy on slower devices...

Reply:
fixed